mrbelvedereci/api/serializers/build.py

```diff
@@ -0,0 +1,105 @@
+from rest_framework import serializers
+from mrbelvedereci.api.serializers.cumulusci import OrgSerializer
+from mrbelvedereci.api.serializers.cumulusci import ScratchOrgInstanceSerializer
+from mrbelvedereci.api.serializers.repository import BranchSerializer
+from mrbelvedereci.api.serializers.repository import RepositorySerializer
+from mrbelvedereci.api.serializers.plan import PlanSerializer
+from mrbelvedereci.build.models import Build
+from mrbelvedereci.build.models import BuildFlow
+from mrbelvedereci.build.models import Rebuild
+from mrbelvedereci.cumulusci.models import Org
+from mrbelvedereci.plan.models import Plan
+from mrbelvedereci.repository.models import Branch
+from mrbelvedereci.repository.models import Repository
+
+class BuildFlowSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = BuildFlow
+        fields = (
+            'id',
+            'build',
+            'error_message',
+            'exception',
+            'flow',
+            'rebuild',
+            'status',
+            'tests_fail',
+            'tests_pass',
+            'tests_total',
+            'time_end',
+            'time_queue',
+            'time_start',
+        )
+        
+class RebuildSerializer(serializers.HyperlinkedModelSerializer):
+    org_instance = ScratchOrgInstanceSerializer(read_only=True)
+    class Meta:
+        model = Rebuild
+        fields = (
+            'id',
+            'build',
+            'error_message',
+            'exception',
+            'org_instance',
+            'status',
+            'user',
+            'time_end',
+            'time_queue',
+            'time_start',
+        )
+
+# mrbelvedereci.build Models
+class BuildSerializer(serializers.HyperlinkedModelSerializer):
+    branch = BranchSerializer(read_only=True)
+    branch_id = serializers.PrimaryKeyRelatedField(
+        queryset=Branch.objects.all(),
+        source='branch',
+        write_only=True
+    )
+    flows = BuildFlowSerializer(many=True, read_only=True)
+    org = OrgSerializer(read_only=True)
+    org_id = serializers.PrimaryKeyRelatedField(
+        queryset=Org.objects.all(),
+        source='org',
+        write_only=True
+    )
+    org_instance = ScratchOrgInstanceSerializer(read_only=True)
+    plan = PlanSerializer(read_only=True)
+    plan_id = serializers.PrimaryKeyRelatedField(
+        queryset=Plan.objects.all(),
+        source='plan',
+        write_only=True
+    )
+    repo = RepositorySerializer(read_only=True)
+    repo_id = serializers.PrimaryKeyRelatedField(
+        queryset=Repository.objects.all(),
```

Comment:
this manager isn't scoped to running user right? could this leak the names of our private repos if we opened up the API?

Reply:
That's correct.  For now, the API is configured to allow full access to is_staff users.  We'll work on more granular permissions in a future PR, sometime very soon though.